packages/fxa-settings/src/pages/Signin/utils.test.tsx

```diff
@@ -0,0 +1,197 @@
+import { InMemoryCache } from '@apollo/client';
```

Comment:
Mingyuan made us a new test utils file, `pages/Signin/utils.test.ts`, we should probably update that one instead? (This was very recent)